### DISCUSSION TITLE
Failed to proxy http://localhost:3005/ Error: connect ECONNREFUSED ::1:3005

### DISCUSSION THREAD

**Discussion by athar-qadri** [supabase/supabase#16746] [General]
I am facing below error when running the development locally.

www:dev: Failed to proxy http://localhost:3005/ Error: connect ECONNREFUSED ::1:3005
www:dev:     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1495:16) {
www:dev:   errno: -61,
www:dev:   code: 'ECONNREFUSED',
www:dev:   syscall: 'connect',
www:dev:   address: '::1',
www:dev:   port: 3005
www:dev: }

**Comment by emildahjeremiah-dotcom** [comment]
rectify
